examples/stdio/main.c

```diff
@@ -14,12 +14,13 @@
  * limitations under the License.
  */
 
+#include <stdio.h>
 #include "greentea-client/test_env.h"
 
 int main()
 {
     const char *key = "demo_key";
-    const int value = 60;
+    const char *value = "60";
```

Comment:
Why the change to a string? Shall we explain in a standalone commit's message?

Reply:
I converted this example to C, so overloads of `greentea_send_kv` are unavailable and the C version of it takes a string.

Maybe I should convert it back to C++, to allow demonstration of the full API?

Reply:
Done, I've converted the files back to C++

Reply:
I didn't pick up that C only allowed strings. I am okay with or without this change, just we need a commit message to explain it if we want to make the change.

Reply:
I've already converted back it to C++ because it allows us to use the full API. Maybe in the future we can send several KV pairs of different types (string, int, multiple ints for reporting passes & failures, ...) in our examples to demonstrate this. So does the current state of the PR look okay?

Reply:
Yup, all fine.